hugr-passes/src/monomorphize.rs

```diff
@@ -318,7 +320,7 @@ fn write_type_arg_str(arg: &TypeArg, f: &mut std::fmt::Formatter<'_>) -> std::fm
 ///    simple obvious thing.
 ///  - For all TypeArg Constructors we choose a short prefix (e.g. `t` for type)
 ///    and use "t({arg})" as the string representation of that arg.
-fn mangle_name(name: &str, type_args: impl AsRef<[TypeArg]>) -> String {
+pub fn mangle_name(name: &str, type_args: impl AsRef<[TypeArg]>) -> String {
```

Comment:
The docstring could be improved for public consumption:
- The bullet list isn't grammatically coherent. I suggest making each bullet a sentence, punctuated as such.
- Is the first bullet actually right? It doesn't seem to match the implementation below.
- What is the "simple obvious thing"?